src/openrct2/world/TileElement.h

```diff
@@ -199,15 +199,17 @@ struct PathElement : TileElementBase
     uint8_t Edges;          // 8
     uint8_t Flags2;         // 9
     uint8_t SlopeDirection; // 10
+
     union
     {
         uint8_t AdditionStatus; // 11, only used for litter bins
         ride_id_t rideIndex;    // 11
     };
     ::StationIndex StationIndex; // 13
+    uint8_t isWideFlags;         // 14
```

Comment:
```suggestion
    uint8_t WideFlags;         // 14
```
I think the "is" should be dropped. "is..." variables usually imply boolean value.